src/blog/2020-05-15/azure-pipelines-integration.pug

```diff
@@ -0,0 +1,158 @@
+extends ../../layout/blog-layout.pug
+block content
+    :markdown-it
+        When using continuous integration it can be hard to keep track of your test quality. By integrating Stryker in your build pipelines you can ensure your test quality stays at the same level. No matter how many changes your code gets to see.
+
+        To reach this point we had to improve a few points:  
+        [🚀 Speed](#speed)  
+        [🔎 Git integration](#git-integration)  
+        [✨ Reporting](#reporting)  
+        [⛔ Forcing test quality](#forcing-test-quality)  
+        
+        ## 🚀 Speed 
+
+        Stryker.NET is now faster than ever! This means you can run mutation testing for every pull request without waiting too long for your build to succeed.
+
+        Some performance improvements we have implemented:
+        - Mutation switching
+            - We compile all mutants at once using conditional statements around the mutations. When testing the mutants we only have to flip the switch to turn in on 🔛
+        - Integration with VSTest
+            - This is the fastest way to run your tests. And VSTest supports all well known test frameworks.
+        - Coverage analysis
+            - We do not test mutants that are not covered by any test. And we only run the tests that cover a mutant when testing the mutant.
+        - Testing mutants in parallel
+            - We run multiple test sessions at once to test the mutants even faster.
+        - Using the same test session for non overlapping mutants
+            - Mutants that are in a different file and have no overlapping covered tests are run in the same test session to remove the overhead of starting a new test session.
+
+        By implementing all these performance tweaks we can now proudly say we are fast enough to use in your pipeline! So if performance was holding you back, now is the moment to give Stryker.NET a try! 🚀
+        
+        ## 🔎 Git integration
+
+        On large projects we can also use git to filter changed files in a pull request. That way only the changed code will be mutated. This helps to use mutation testing in pull request even on the largest projects.
+
+        How does this work? Simply run Stryker.NET with the following command in your PR build:
+
+        ```
+        dotnet stryker --diff
+        ```
+
+        This will use the master branch as base by default. If you use a different branch as base for your pull requests you can change the base branch like this:
+
+        ```
+        dotnet stryker --diff --git-source "branchname"
```

Comment:
Voorbeeld van hoe je dat in azure pipelines zou doen met environment variables die beschikbaar zijn in de build